code/game/objects/effects/decals/turfdecal/tilecoloring.dm

```diff
@@ -173,7 +187,6 @@
 
 /obj/effect/turf_decal/tile/white
 	name = "white tile decal"
-	icon_state = "corner_white"
```

Comment:
?

Reply:
op - "as well as an incorrect icon_state on one of the subtypes."

Incorrect var that wasn't corrected in the original PR.